src-tauri/src/utils.rs

```diff
@@ -244,8 +244,13 @@ pub fn spawn_stats_thread(
                         }
                     }
                 }
+                Ok(None) => {
+                    info!("gRPC stream has been closed");
```

Comment:
@moubctez we should inform clearly: gRPC stream to the defguard-service managing connections has been closed. The message is to generic.